py2many/rewriters.py

```diff
@@ -302,6 +302,10 @@ def visit_Compare(self, node):
                     ret = ast.parse("a.find(b) != string.npos").body[0].value
                     ret.left.func.value = right
                     ret.left.args[0] = left
+                elif self._language == "python":
```

Comment:
Why not short circuit on line 283?

Reply:
I was thinking that it might be better for the "simplified" Python to be using `str.find`, as that is easier to detect, and is able to be handled with the plugins system.  In many languages there is an equivalent function which returns `-1`.

Anyway, a short circuit on line 283 is better for now.